test/testdata/resolver/requires_ancestor_disabled.rb

```diff
@@ -0,0 +1,44 @@
+# typed: true
+# enable-experimental-requires-ancestor: false
+
+### Does not suggest helper module if disabled
+
+module Test1
+  module M1
+    requires_ancestor M2 # error: Method `requires_ancestor` does not exist on `T.class_of(Test1::M1)`
+  end
+
+  module M2; end
+end
+
+### Does not resolve methods if disabled
+
+module Test2
+  module M1
+    extend T::Helpers
+    requires_ancestor M2
```

Comment:
Why doesn't this raise an error when the feature is disabled?

Reply:
The problem is the definition is coming from the Helpers RBi.

I previously added a check to raise an error if the method was used without the option.

You asked me to remove it here, and I raised the same concerns two weeks ago https://github.com/sorbet/sorbet/pull/3409#discussion_r568922403.

Reply:
I was wondering after I left this comment if it was missing because I had asked you to remove it, thanks! 😅